### ISSUE TITLE
Webview Font Size in Android using One UI 6.1 on Samsung Flip

### ISSUE THREAD

**Issue by RiverboatGambler** [OneUIProject/sesl#18]
From what I can determine, there is an issue in the One UI 6.1 that causes the normal font to be rendered extremely small in an Android Webview. Has anyone else noticed this? 

Did the default font or font-family for an HTML resource asset get changed?

I'm attaching two screenshots.

One is from a Galaxy 14; the other from the Galaxy Flip 3 test device that I purchased when a user reported this issue. Both devices are running Android 14 with OneUI 6.1 and no additional updates are available for either.

The Flip came from Amazon with Android 12 installed. It wasn't until I fully upgraded from there to Android 14/OneUI 6.1 that I was able to reproduce the issue. 

The content in the webview (main part of the screen) is an embedded HTML resource that contains very basic markup

<img loading="lazy" src="https://github.com/user-attachments/assets/2c2c620b-3020-45e2-aed1-004529fdc726" width="30%"/>
<img loading="lazy" src="https://github.com/user-attachments/assets/31df3e1c-99c7-4560-8213-67701835bd91" width="30%"/>

**Comment by ronaldoguedess** [comment]
please, show an screnshot

**Comment by RiverboatGambler** [comment]
I attached some images. Apologies for how large they are! Holy Cow! If there's a way to shrink, them, please advise and I will. I also updated the original post with additional diagnostic details.

**Comment by salvogiangri** [comment]
> I attached some images. Apologies for how large they are! Holy Cow! If there's a way to shrink, them, please advise and I will. I also updated the original post with additional diagnostic details.

Updated your issue main post to properly resize the images. May I know how is this related to the libs? Did you also try the vanilla androidx ones?

**Comment by RiverboatGambler** [comment]
I don't know that I can lay this at the feet of any specific libraries. I'm not specifically interacting with any libraries outside of how they are used by the host devices.

I had users report this issue to me on their Galaxy Flip 3 devices. I could not reproduce the problem so I purchased a Galaxy Flip 3 from Amazon. 

The device I received had Android 12 installed on it and the issue did present; Unfortunately, I didn't make a note of the OneUI version when the problem first presented.

After going through a series of upgrades to Android 14. OneUI 6.0, the issue went away. 

After completing all available upgrades I ended up on Android 14, OneUI 6.1 and the issue reappeared. 

Given that my other Galaxy 14 phones are also at Android 14, OneUI 6.1 and are not presenting this issue, my assumption is that the issue is related to how the Webview control is being supported in OneUI 6.1 and, perhaps, specifically by the Flip 3 device. 

Just a theory. 

I am currently trying to find some workarounds. In my amended original post, I added that the content being displayed in the Webview in the center of the screen are embedded HTML files using basic markup. They do not define a font-family and the default font-size is defined as normal
